src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -247,6 +262,8 @@ private Delegate GetGetterCore(DataViewRow input, int iinfo, out Action disposer
             var srcType = input.Schema[srcCol].Type;
             if (!(srcType is VectorDataViewType vectorType))
                 return ComposeGetterOne(input, iinfo, srcCol, srcType);
+            if (_columns[iinfo].Combine)
+                return ComposeGetterCombined(input, iinfo, srcCol, vectorType);
             return ComposeGetterVec(input, iinfo, srcCol, vectorType);
```

Comment:
We need to figure out the onnx export mechanism for this.

Reply:
For now I'm making this case non-exportable.

---
In reply to: [386741545](https://github.com/dotnet/machinelearning/pull/4828#discussion_r386741545) [](ancestors = 386741545)